README.md

```diff
@@ -1,72 +1,59 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# HTML/CSS & JavaScript capstone project - D'GEES InfoTech Global Systems Summit page
 
+  > This Project is about creating a responsive webpage
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- HTML
+- CSS
+- Javascript
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo 
 
+[Live Demo Link](https://drive.google.com/file/d/13Km0UwaQjy5rvKNykDjiykMde0BPqfpX/view?)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+-  To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+- Basic HTML & CSS knowledge
+- Basic JavaScript knowledge
+
 ### Setup
 
-### Install
+- git clone github.com/ecekpo/emem-capstone-proj.git
 
 ### Usage
 
-### Run tests
-
-### Deployment
-
-
+- Switch to emem-capstone-proj directory
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+👤 **Emem Ekpo**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@git@ecekpo](https://github.com/@ecekpo)
+- Twitter: [@twitterememcookey](https://twitter.com/ememcookey)
+- LinkedIn: [LinkedIn](https://https://www.linkedin.com/in/emem-ekpo-857135234/)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/@ecekpo/issues/).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
```

Comment:
-  Kindly make sure that the README or Licence contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).